ui/lib/core/addon/templates/components/replication-action-disable.hbs

```diff
@@ -37,8 +37,8 @@
   @testSelector="disable"
 >
   <p class="has-bottom-margin-m">
-    {{#if (and model.replicationAttrs.isPrimary (eq replicationDisplayMode "DR"))}}This cannot be undone. {{/if}}
-    Disabling {{replicationDisplayMode}} Replication entirely on this {{if (eq model.replicationAttrs.isPrimary true) "primary" "secondary"}} cluster means that:
+    {{#if (and model.replicationAttrs.isPrimary (eq model.replicationModeForDisplay "Disaster Recovery"))}}This cannot be undone. {{/if}}
```

Comment:
This line probably just needs to be updated to `... (eq model.replicationMode "dr")...`